ui/src/pages/GlobalHome.js

```diff
@@ -0,0 +1,293 @@
+    this.state = {
+      isLoaded: false,
+      error: null,
+      repos: []
+    };
+  }
+
+  componentDidMount() {
+    fetch("/global/d/repoinfo")
+      .then(res => res.json())
+      .then(
+        (repos) => {
+          for (var i = 0; i < repos.length; i++) {
+            repos[i]['displayCat'] = i % 5;
+          }
+          this.setState({
+            isLoaded: true,
+            repos: repos
+          });
+        },
+        (error) => {
+          this.setState({
+            isLoaded: true,
+            error: error
+          });
+        }
+      );
+  }
+
+  renderHeader() {
+    return (
+      <div>
+        <h1 className="mdc-typography--headline1">
+          <img src="/static/icons/maticon_person_pin.svg" />
+          <FormattedHTMLMessage {...messages.productNameWithEmphasis} />
+        </h1>
+        <p className="mdc-typography--body1 globalhome-headerdesc">
+          <FormattedMessage {...messages.tagline} />
+        </p>
+        <Button
+          className="pf-button-secondary globalhome-howsitworkbutton"
+          raised
+          onClick={() => console.log('clicked!')}
+        >
+          {this.props.intl.formatMessage(messages.howDoesItWork)}
+        </Button>
+      </div>
+    );
+  }
+
+  renderRespondersCard() {
+    return (
+      <Card className="globalhome-thirdpartycard">
+        <div>
+          <h3 className="mdc-typography--headline3">
+            <FormattedMessage {...messages.responders} />
+          </h3>
+          <p className="mdc-typography--body1">
+            <FormattedHTMLMessage {...messages.respondersHowToHelp} />
+          </p>
+        </div>
+        <CardActions>
+          <CardActionButtons>
+            <Button
+              className="pf-button-secondary"
+              raised
+              onClick={() => console.log('clicked!')}
+            >
+              {this.props.intl.formatMessage(messages.learnHow)}
+            </Button>
+          </CardActionButtons>
+        </CardActions>
+      </Card>
+    );
+  }
+
+  renderDevelopersCard() {
+    return (
+      <Card className="globalhome-thirdpartycard">
+        <div>
+          <h3 className="mdc-typography--headline3">
+            <FormattedMessage {...messages.developers} />
+          </h3>
+          <p className="mdc-typography--body1">
+            <FormattedHTMLMessage {...messages.developersHowToHelp} />
+          </p>
+        </div>
+        <CardActions>
+          <CardActionButtons>
+            <Button
+              className="pf-button-secondary"
+              raised
+              onClick={() => console.log('clicked!')}
+            >
+              {this.props.intl.formatMessage(messages.getStarted)}
+            </Button>
+          </CardActionButtons>
+        </CardActions>
+      </Card>
+    );
+  }
+
+  renderThirdPartyHelpCards() {
+    return (
+      <div className="globalhome-thirdpartywrapper">
+        <div className="endbars-headerline-wrapper" dir="ltr">
```

Comment:
Why is it hard-coded to "ltr"?

Reply:
More discussion on PR #472, but in short it's to make the CSS work without using CSS properties that aren't widely supported.